src/api/tests/test_protocol_validations.py

```diff
@@ -1,242 +1,125 @@
-import copy
```

Comment:
I should know this, but -- I gather fixtures are automatically imported/processed/scoped based on a `fixtures` subdir name, so you don't have to import them here?

Reply:
yup